skimage/draw/__init__.py

```diff
@@ -1,3 +1,51 @@
+"""Utilities for generating coordinates of different shapes and curves.
+
+Use this module for generating coordinates of a bezier curve, circle, ellipse, rectangle, 
+generating ellipsoid, calculating surface-area and volume of ellipsoid, 
+drawing lines, setting pixel color at a coordinate.
+
+...
+
+Using this module following functionalities can be accessed
+
+
+bezier_curve(r0, c0, r1, c1, ...)     Generate Bezier curve coordinates.
+
+circle_perimeter(r, c, radius)        Generate circle perimeter coordinates.
+
+circle_perimeter_aa(r, c, radius)     Generate anti-aliased circle perimeter coordinates.
+
+disk(center, radius, *[, shape])      Generate coordinates of pixels within circle.
+
+ellipse(r, c, r_radius, c_radius)     Generate coordinates of pixels within ellipse.
+
+ellipse_perimeter(r, c, ...[, ...])   Generate ellipse perimeter coordinates.
+
+ellipsoid(a, b, c[, spacing, ...])    Generates ellipsoid with semimajor axes aligned with grid dimensions on grid with specified spacing.
+
+ellipsoid_stats(a, b, c)              Calculates analytical surface area and volume for ellipsoid with semimajor axes aligned with grid dimensions of specified spacing.
+
+line(r0, c0, r1, c1)                  Generate line pixel coordinates.
+
+line_aa(r0, c0, r1, c1)               Generate anti-aliased line pixel coordinates.
+
+line_nd(start, stop, *[, ...])        Draw a single-pixel thick line in n dimensions.
+
+polygon(r, c[, shape])                Generate coordinates of pixels within polygon.
+
+polygon2mask(image_shape, polygon)    Compute a mask from polygon.
+
+polygon_perimeter(r, c[, ...])        Generate polygon perimeter coordinates.
+
+random_shapes(image_shape, ...)       Generate an image with random shapes, labeled with bounding boxes.
+
+rectangle(start[, end, extent, ...])  Generate coordinates of pixels within a rectangle.
+
+rectangle_perimeter(start[, ...])     Generate coordinates of pixels that are exactly around a rectangle.
+
+set_color(image, coords, color)       Set pixel color in the image at the given coordinates.
```

Comment:
Why did you decide to add this part? Do you think it duplicates the [automatically generated function list in our HTML](https://scikit-image.org/docs/dev/api/skimage.draw.html)?

Reply:
I did this because in spite of duplication, most of the time a developer would want to know what arguments can go in a corresponding function, instead of checking with a website by switching tabs, I thought if he had the opportunity to make a quick glance within the IDE, the docstrings would provide more value.

Reply:
I follow your argument and for some developer workflows this might be a benefit. However, there is a significant maintenance cost for the change you propose. Each time, we change the API in that module we have to update this list by hand. And if we forget, then those docstrings are actually misleading developers! 

That alone is a trade-off I would prefer not to make. Additionally, this docstring is also part of our website. You can see how the change is rendered in HTML [here](https://output.circle-artifacts.com/output/job/6e198963-d465-428c-8d9e-f82f6dae9145/artifacts/0/doc/build/html/api/skimage.draw.html).